packages/web/src/views/governance/components/proposals-list/view-proposal-modal/ViewProposalModal.tsx

```diff
@@ -49,163 +39,138 @@ const ViewProposalModal: React.FC<ViewProposalModalProps> = ({
-    };
-  }, [modalRef]);
 
   if (!proposalDetail) return null;
 
   return (
-    <>
-      <ViewProposalModalBackground>
-        <ViewProposalModalWrapper ref={modalRef}>
-          <div className="modal-body">
-            <ModalHeaderWrapper>
-              <div className="header">
-                <div className="title">
-                  <span>{`#${proposalDetail.id} ${proposalDetail.title}`}</span>
-                  {breakpoint !== DEVICE_TYPE.MOBILE && (
-                    <TypeBadge type={proposalDetail.type} />
-                  )}
-                </div>
-                <div
-                  className="close-wrap"
-                  onClick={() => setSelectedProposalId(0)}
-                >
-                  <IconClose className="close-icon" />
-                </div>
-              </div>
-              {breakpoint === DEVICE_TYPE.MOBILE && (
+    <Modal>
+      <div className="modal-body">
+        <ModalHeaderWrapper>
+          <div className="header">
+            <div className="title">
+              <span>{`#${proposalDetail.id} ${proposalDetail.title}`}</span>
+              {breakpoint !== DEVICE_TYPE.MOBILE && (
                 <TypeBadge type={proposalDetail.type} />
               )}
-              <div className="active-wrapper">
-                <StatusBadge
-                  status={proposalDetail.status}
-                  time={proposalDetail.time}
-                />
-              </div>
-            </ModalHeaderWrapper>
-            <ProposalContentWrapper>
-              <div className="content">
-                {proposalDetail.type === "COMMUNITY_POOL_SPEND" && (
-                  <>
-                    <div className="variable">
-                      <div className="variable-type">
-                        {t("Governance:detailModal.content.recipient")}
-                      </div>
-                      {proposalDetail.content.recipient}
-                    </div>
-                    <div className="variable">
-                      <div className="variable-type">
-                        {t("Governance:detailModal.content.amount")}
-                      </div>
-                      {proposalDetail.content.amount}
-                    </div>
-                  </>
-                )}
-                {proposalDetail.type === "PARAMETER_CHANGE" && (
-                  <div className="variable">
-                    <div className="variable-type">
-                      {t("Governance:detailModal.content.change")}
-                    </div>
-                    {proposalDetail.content.parameters?.map((item, index) => (
-                      <div key={index}>
-                        {`Subspace: "${item.subspace}", Key: "${item.key}", Value: "${item.value}"`}
-                      </div>
-                    ))}
+            </div>
+            <div
+              className="close-wrap"
+              onClick={() => setSelectedProposalId(0)}
```

Comment:
Does it close when the proposal id is 0 ?
It would be nice to be more explicit.

Reply:
good point, I think we could pass `closeModal` instead.
I'll work on it in next PR